examples/remarketing/add_flexible_rule_user_list.rb

```diff
@@ -28,10 +28,22 @@ def add_combined_rule_user_list(customer_id)
   # ENV['HOME']/google_ads_config.rb when called without parameters
   client = Google::Ads::GoogleAds::GoogleAdsClient.new
 
+  user_visited_site1_rule_info = create_user_list_rule_info_from_url(
+    client,
+    'http://example.com/example1',
+  )
+  user_visited_site2_rule_info = create_user_list_rule_info_from_url(
+    client,
+    'http://example.com/example2',
+  )
+  user_visited_site3_rule_info = create_user_list_rule_info_from_url(
+    client,
+    'http://example.com/example3',
+  )
+
   # Creates a user list.
   operation = client.operation.create_resource.user_list do |u|
-    u.name = "All visitors to http://example.com/example1 AND " \
-      "http://example.com/example2 ##{(Time.new.to_f * 1000).to_i}"
+    u.name = "Flexible rule user list for example.com ##{(Time.new.to_f * 1000).to_i}"
     u.description = "Visitors of both http://example.com/example1 AND " \
```

Comment:
Missing:

` but NOT http://example.com/example3`

See http://tg/1717759 and follow-up http://tg/1742890.